packages/auth/src/core/errors.ts

```diff
@@ -603,5 +603,6 @@ export const AUTH_ERROR_CODES_MAP_DO_NOT_USE_INTERNALLY = {
   MISSING_RECAPTCHA_VERSION: 'auth/missing-recaptcha-version',
   INVALID_RECAPTCHA_VERSION: 'auth/invalid-recaptcha-version',
   INVALID_REQ_TYPE: 'auth/invalid-req-type',
-  INVALID_HOSTING_LINK_DOMAIN: 'auth/invalid-hosting-link-domain'
+  INVALID_HOSTING_LINK_DOMAIN: 'auth/invalid-hosting-link-domain',
+  MISSING_PASSWORD: 'auth/missing-password'
```

Comment:
The properties of this object closely mirror the internal `AuthErrorCode` enum above. To keep the ordering consistent, can you move this new property below `MISSING_PHONE_NUMBER` on line 562?
Here's the ordering in the internal enum:
https://github.com/firebase/firebase-js-sdk/blob/ccbf7ba36fa9b10c100296489f8be265c8f83e41/packages/auth/src/core/errors.ts#L89-L95